trpc/router/mailbox/conversations/index.ts

```
@@ -169,15 +169,40 @@ export const conversationsRouter = {
+
+      let filterInput;
+      if (Array.isArray(conversationFilter)) {
+        const assignee = prevAssigneeId ? [prevAssigneeId as string] : [];
+        filterInput = {
+          limit: 25,
+          assignee,
+          status: input.status !== undefined ? [input.status] : undefined,
+        };
+      } else {
+        const assignee = prevAssigneeId ? [prevAssigneeId as string] : [];
+        filterInput = {
```

ESLint

[error] 'filterInput' is assigned a value but never used. Allowed unused vars must match /^_/u.

(@typescript-eslint/no-unused-vars)